boards/arm/efm32wg_stk3800/board.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Christian Taedcke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <board.h>
+#include <gpio.h>
+
+static int efm32wg_stk3800_init(struct device *dev)
+{
+	struct device *bce_dev; /* Board Controller Enable Gpio Device */
```

Comment:
well, you could use the dev variable already existing on the stack instead of allocating another one (and thus, not need of ARG_UNUSED())

Reply:
I would prefer not to reuse function parameter, just to safe some stack and because they are they are by chance the same type.